dojo/utils.py

```diff
@@ -1616,7 +1608,7 @@ def set_default_notifications(sender, instance, created, **kwargs):
 
 
 @receiver(post_save, sender=Engagement)
-def engagement_post_Save(sender, instance, created, **kwargs):
+def engagement_post_Save(sender, instance, created, **_kwargs):
```

Comment:
why using _kwargs instead of kwargs what is used everywhere else?

Reply:
Since it's unused, the underscore tells the linter to ignore it.

Reply:
linting removed from PR, I'll offer something separate for that